requirements/testing.txt

```diff
@@ -8,7 +8,7 @@ pytest==4.1.1  # pyup: <4.2
 # probably a bug related to overriding django settings in
 # previous tests
 pytest-django==3.4.6  # pyup: <3.4.7
-pytest-xdist==1.27.0
+pytest-xdist==1.28.0
```

Comment:
We need to update pytest https://github.com/pytest-dev/pytest-xdist/blob/master/CHANGELOG.rst#features, we need to check the corporate site, there was an incompatibility there with the version of pytest